### ISSUE TITLE
Can pinpoint-web show requests by specific path?

### ISSUE THREAD

**Issue by pengsili** [naver/pinpoint#6146]
Dear support,
   In /#/main page. i can draw a frame in the chart  to choose requests which i want to do further analyze. But i don`t know how to choose request by specific path.For example, i want to see all requests whore path is /service/user during the time duration i choose. Does pinpoint-web support this feature? if yes, please share how to do it.
Thanks a lot

**Comment by Xylus** [comment]
Hi @pengsili 
You can right-click on the edge pointing to your node to select "Transaction Wizard" to filter requests by specific path or pattern-matched path.

**Comment by pengsili** [comment]
Hi @Xylus ,
  Thanks for your reply. But seems the search result not only includes the url i input.For example, i filter by /service/user. The result includes some other requests which have different path.

**Comment by koo-taejin** [comment]
@pengsili 

Please capture and post the execution screen and the results screen.
In addition, post the web module log when the problem occurs.

thanks. :)

**Comment by pengsili** [comment]
Hi koo-taejin
 Thanks for your reply. Here is the filter wizard screen shot and the filter result screen shot.
![Filter Wizard](https://user-images.githubusercontent.com/22196529/68930977-825ad300-07ca-11ea-9af2-ff7d7ddbe43b.png)
![Result](https://user-images.githubusercontent.com/22196529/68930983-84bd2d00-07ca-11ea-9b07-2767d4dca877.png)

Thanks

**Comment by koo-taejin** [comment]
@pengsili 

As mentioned above
Please post the log of the web module at the time of the filter check.
It's even better if you record it to Debug Level. :)

thanks.

**Comment by pengsili** [comment]
Hi Koo-taejin,
    We use pinpoin in docker. And here is part of log which i got by command 'docker logs pinpoint-web'.
Thanks
[pinpoint-web.log](https://github.com/naver/pinpoint/files/3873105/pinpoint-web.log)

**Comment by koo-taejin** [comment]
hi @pengsili 

I checked log you posted.

Look at the log
It appears to have searched for `/ wls_prs_sps / action / router / SearchOrderController_searchSr`

As a result
```
TransactionId {agentId = 'sps-2', agentStartTime = 1574270106626, transactionSequence = 934}, rpc = '/ wls_prs_sps / action / router / SearchOrderController_searchSr'
TransactionId {agentId = 'sps-2', agentStartTime = 1574270106626, transactionSequence = 909}, rpc = '/ wls_prs_sps / action / router / SearchOrderController_searchSr'
TransactionId {agentId = 'sps-1', agentStartTime = 1574269634976, transactionSequence = 891}, rpc = '/ wls_prs_sps / action / router / SearchOrderController_searchSr'
```

It seems that these three transactions have been filtered.
There seems to be no problem, but please check once you have sent the log of the filtered success situation.

thanks. :)

**Comment by pengsili** [comment]
Hi @koo-taejin 
 Sorry for late reply. I found that we encoded an object and added to url. Such as the second transaction in the picture above. In this case,should this transaction not be shown here?Since its path was that the same with that i input.
Thanks

**Comment by koo-taejin** [comment]
hi @pengsili 

Oh I seems to know the cause.

Perhaps the request was handled this way.

```
TransactionId (sps-2:1574270106626:934)
 1. tms-qa:sps-2, ' /wls_prs_sps/action/router/SearchOrderController_searchSr 
	-> call localhost 
		2.tms-qa:sps-2, /wls_dom_sps/restful/order/searchOrder/%7B%22start%22%3A0%2C%22limit%22%3A100%2C%22startPage%22%3A1%2C%22page%22%3A0%2C%22miniModel%22%3Afalse....
			-> call remote 
				3. 'tms-qa-ath:ath-1', /wls_auth/service/AuthorizationService/userInfoWithPermission
	-> call remote 
		4. 'tms-qa-ath:ath-1',  '/wls_auth/service/AuthorizationService/PermissionToken' 
	-> call remote 
		5. 'tms-qa-ath:ath-1',  '/wls_auth/service/AuthorizationService/PermissionToken'
	-> call remote 
		6. 'tms-qa-ath:ath-1', '/wls_auth/service/AuthorizationService/PermissionToken'
```

Every transaction is made up of  one or more spans.
Each Span often contains information such as a url.

However, if you look at the Distributed Call Stack, it is divided by location.

For example
If you query the 'sps-2: 1574270106626: 934' transaction in the tms-qa application, numbers 1 and 2 will be shown in Distributed CallTree.
However, if you query the same transaction in the tms-qa-ath application, you would see information 3, 4, 5, and 6 in Distributed CallTree.

In other words, if we do `/wls_prs_sps/action/router/SearchOrderController_searchSr` on the Filter, Pinpoint will return the TransactionId associated with it.
However, in this case, you have request that called localhost, so you will have 1 and 2 in one TransactionId.

This is definitely the situation we intended, But  I wonder a little bit about whether this form is good in the situation you mentioned.

I will think about this a bit more.

thanks :)

**Comment by pengsili** [comment]
Hi @koo-taejin 
Thanks for your reply. I think i understand it now. And i agree with you that this feature may not fit for our use case.
